test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -146,7 +146,7 @@ string SolidityExecutionFramework::addPreamble(string const& _sourceCode)
 {
 	// Silence compiler version warning
 	string preamble = "pragma solidity >=0.0;\n";
-	if (_sourceCode.find("// SPDX-License-Identifier:") == string::npos)
+	if (_sourceCode.find("SPDX-License-Identifier:") == string::npos)
```

Comment:
Why not add the license type as a `string` parameter to `withPreamble`, make it a helper free function and also call it here?

Reply:
I don't think we really need that in practice. We should use the same license in all preambles.

Reply:
Well, maybe. The reason I did not do that yet, is that I did not have a good place to put it and for analysis-based tests `AnalysisFramework` seemed good enough. And I'm going to be doing a task related to execution framework next anyway so I decided it's good enough for now. I can refactor it further when I get to it.

I guess I could also make it a free function like you suggest and create some kind of `Common.h`...

Reply:
Ah ok, yeah, I would prefer to at least have it as free function already then, but fine to postpone such refactor :)

Reply:
Wouldn't make sense to move it to `libsolutil`?

Reply:
It's purely a test utility. We never need something like this in the compiler itself so it does not feel right to put it there.